src/cider/nrepl/middleware/info.clj

```diff
@@ -66,12 +66,15 @@
   [{:keys [ns symbol class member] :as msg}]
   (let [[ns symbol class member] (map u/as-sym [ns symbol class member])]
     (if-let [cljs-env (cljs/grab-cljs-env msg)]
+      (info-cljs cljs-env symbol ns)
```

Comment:
Did you intend to only remove the `(handle-cljx-sources ...)` call ?

Reply:
Ops, my bad!